lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -258,7 +258,7 @@ def show_versions(self, trans: GalaxyWebTransaction, workflow_id, **kwds):
 
         Lists all versions of this workflow.
         """
-        stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id, **kwds)
+        stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id)
```

Comment:
```suggestion
        instance = util.string_as_bool(kwd.get("instance", "false"))
        stored_workflow = self.workflow_manager.get_stored_accessible_workflow(trans, workflow_id, by_stored_id=not instance)
```

Reply:
I think this would be correct.

Reply:
Is this change still relevant now that this should be fixed in bioblend?

Reply:
I think it is, the docstring documents an `instance` param, but this doesn't work currently. And any other `kwd` will also cause an error.

Reply:
Okay, thanks for clarifying